usr/share/rear/build/GNU/Linux/600_verify_and_adjust_udev.sh

```diff
@@ -6,8 +6,10 @@ test -d $ROOTFS_DIR/etc/udev/rules.d || return 0
 
 # check systemd version
 systemd_version="$(systemd-notify --version 2>/dev/null | grep systemd | awk '{print $2}')"
-[[ -z "$systemd_version" ]] && systemd_version=0
-if version_newer "$systemd_version" 190; then
+if [[ -n "$systemd_version" ]] ; then
+   if ! version_newer "$systemd_version" 190 ; then
+       Error "systemd version is '$systemd_version', systemd versions below 190 are not supported"
+   fi
    ps ax | grep -v grep | grep -q systemd-udevd && { # check if daemon is actually running
```

Comment:
Just a thought: Isn't `pgrep` universally available?

Reply:
I dislike this line, but I wanted to keep the change minimal.

Reply:
> Just a thought: Isn't `pgrep` universally available?

Indeed even RHEL 6 has `pkill`, and in the same package as `ps`:
```
[root@pcahyna-1mt-rhel-6 ~]# rpm -qf $(command -v pkill)
procps-3.2.8-45.el6_9.3.x86_64
[root@pcahyna-1mt-rhel-6 ~]# rpm -qf $(command -v ps)
procps-3.2.8-45.el6_9.3.x86_64
```

Reply:
Also on SLES12-SP5 pgrep pkill and ps
belong to the same RPM binary package
procps-3.3.9-11.18.1.x86_64
```
# type -a pgrep
pgrep is /usr/bin/pgrep
pgrep is /bin/pgrep
pgrep is /usr/bin/X11/pgrep

# rpm -qf /usr/bin/pgrep
procps-3.3.9-11.18.1.x86_64

# type -a pkill
pkill is /usr/bin/pkill
pkill is /bin/pkill
pkill is /usr/bin/X11/pkill

# rpm -qf /usr/bin/pkill
procps-3.3.9-11.18.1.x86_64

# type -a ps
ps is /usr/bin/ps
ps is /bin/ps
ps is /usr/bin/X11/ps

# rpm -qf /usr/bin/ps
procps-3.3.9-11.18.1.x86_64
```